### DISCUSSION TITLE
PathLayer screenspace offset in pixel ?

### DISCUSSION THREAD

**Discussion by vrman** [visgl/deck.gl#7874] [General]
I want to clone my PathLayer with fixed distance offset in every zoom level on map , for example i want to have 40 pixel offset in zoom 16 and also 40 pixel offset at zoom level 2. we can do something like this with PathStyleExtension for dashed lines , but i want to do this on PathLayer self so then it affect on clickable area too.

**Comment by Pessimistress** [comment]
[widthUnits](https://deck.gl/docs/api-reference/layers/path-layer#widthunits)

**Reply by vrman** [reply]
it cant help this, the problem is this bug :
https://github.com/visgl/deck.gl/issues/7875
